gui/threshold_level.py

```diff
@@ -3,18 +3,22 @@
 
 
 class ThresholdLevel:
-    threshold_lvl = None
-
-    def __init__(self, parent_container):
+    def __init__(self, parent_container, bot):
         self.container = parent_container
         self.threshold_lvl = IntVar()
         self.threshold_lvl.set(40)
+        self.bot = bot
 
     def render_threshold_level_widgets(self, r):
         Label(self.container, text="Threshold level % : ") \
             .grid(sticky='e', row=r, column=3)
 
-        OptionMenu(self.container, self.threshold_lvl, *range(40, 101, 10)) \
+        OptionMenu(self.container, self.threshold_lvl,
```

Comment:
lambda doesnt work, simply have a command that accepts one argument and that one argument will automatically be the value of optionmenu